param/parameterized.py

```diff
@@ -1398,11 +1407,14 @@ class Comparator(object):
         numbers.Number: operator.eq,
         basestring: operator.eq,
         bytes: operator.eq,
-        type(None): operator.eq
+        type(None): operator.eq,
     }
+    equalities.update({dtt: operator.eq for dtt in dt_types})
 
     @classmethod
     def is_equal(cls, obj1, obj2):
+        if obj1 is obj2:
+            return True
```

Comment:
While this is of course technically correct it is a major change in behavior. Since complex types cannot be compared properly in many cases this will mean that objects that have been changed inplace now no longer trigger an event. Please open a new issue and let's not mix this change into this PR.